config/crd/bases/osbuilder.project-flotta.io_osbuilds.yaml

```diff
@@ -131,9 +131,17 @@ spec:
                 - targetImage
                 type: object
               kickstart:
-                description: Kickstart may store content of a kickstart file to be
-                  used in the target image
-                type: string
+                description: Kickstart is a reference to a configmap that may store
+                  content of a kickstart file to be used in the target image
+                properties:
+                  name:
+                    description: 'Name of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
+                      TODO: Add other useful fields. apiVersion, kind, uid?'
```

Comment:
this is an auto-generated file by the command `make manifests`
Please edit the comment on top of the kickstart field